types/src/keyless/configuration.rs

```diff
@@ -92,3 +93,8 @@ impl Configuration {
         }
     }
 }
+
+impl OnChainConfig for Configuration {
```

Comment:
I thought this was not gonna work because `Configuration` is using resource groups & there is currently lack of support for on-chain configs on resource groups

Reply:
verified by a smoke test, it works now....